octoprint_octobullet/__init__.py

```diff
@@ -29,10 +33,106 @@ def __init__(self):
 		self._bullet = None
 		self._channel = None
 		self._sender = None
+		self._pc_interval = 5
+		self._message_count = 0
+		self._quiet_time_sec = 1800
+		self._last_message = 0
+		self._etl_format   = "{days:d}:{hours:02d}h {minutes:02d}m"
+		self._eta_strftime = "%H:%M %d-%m"
 
 	def _connect_bullet(self, apikey, channel_name=""):
 		self._bullet, self._sender = self._create_sender(apikey, channel_name)
 
+	#~~ progress message helpers
+	def _sanitize_current_data(self, currentData):
+		if (currentData["progress"]["printTimeLeft"] == None):
+			currentData["progress"]["printTimeLeft"] = currentData["job"]["estimatedPrintTime"]
+		if (currentData["progress"]["printTimeLeft"] == None):
+			self._logger.info("Messasge: Still got no print time {}".format(currentData["progress"]["printTimeLeft"]))
```

Comment:
This and most of the other info logging lines should be `self._logger.debug` and you also had a consistent typo there: "Messasge" instead of "Message" (but why the prefix at all, that it's a log message is fairly visible if it lands in the log anyhow, no need to explicitly state this?).